src/extension/content-script/batteries/Twitter.ts

```diff
@@ -44,12 +44,15 @@ function getUserData(username: string) {
     const location = document.querySelector<HTMLElement>(
       `[data-testid="primaryColumn"] [data-testid="UserLocation"]`
     );
+    const name = document.querySelector<HTMLElement>(
```

Comment:
I am not sure how 100% reliable this is. 
those selectors are always problematic.

Reply:
Should we go for APIs again then?

Reply:
But this should work fine as this fetches the username from here:
![Screenshot from 2022-08-04 22-31-54](https://user-images.githubusercontent.com/64399555/182908754-2ca13eaa-92b3-4975-bbaf-c030b69d5036.png)

Reply:
@reneaaron what do you think?

Reply:
Just had a quick look and couldn't find any better way to do it. Other batteries use selectors as well, so I'd merge it 🤷